framework/src/plugins/TtsPlugin.ts

```diff
@@ -0,0 +1,154 @@
+import { SsmlUtilities } from '@jovotech/common';
+
+import { AudioUtilities, InvalidParentError, JovoResponse } from '..';
+import { Extensible } from '../Extensible';
+import { Jovo } from '../Jovo';
+import { Platform } from '../Platform';
+import { Plugin, PluginConfig } from '../Plugin';
+
+import { Md5 } from 'ts-md5';
+import { TtsCachePlugin } from './TtsCachePlugin';
+import { TtsData } from './TtsData';
+
+export enum TtsTextType {
+  Text = 'text',
+  Ssml = 'ssml',
+}
+
+export interface TtsPluginConfig extends PluginConfig {
+  cache?: TtsCachePlugin;
+  outputFormat: string;
```

Comment:
Could you tell us a bit more about `outputFormat`?

I see this is used once in `ttsResponse = await this.config.cache.getItem(audioKey, locale, this.config.outputFormat);`, but the `getItem` method's third parameter is `fileExtension: string,`

Reply:
Ah, just seeing that you use `outputFormat: 'mp3',` in one example. I'll go ahead and rename `outputFormat` to `fileExtension`, I think that makes it a bit clearer

Reply:
Or am I missing something here? If there's a different between `outputFormat` in `TtsPlugin` and `fileExtension` in `TtsCachePlugin`, we might add a `fileExtension` config option to `TtsCachePlugin` to avoid any confusion.

What do you think @rmtuckerphx?